en/docs/guides/login/logging-into-simplesamlphp-using-is.md

```diff
@@ -0,0 +1,209 @@
+        // The entity ID of the IdP this should SP should contact.  
+            
+        // Can be NULL/unset, in which case the user will be shown a list of available IdPs.  
+            
+        'idp' => 'https://localhost:9443/samlsso',  
+            
+        // The URL to the discovery service.  
+            
+        // Can be NULL/unset, in which case a builtin discovery service will be used.  
+            
+        'discoURL' => NULL,  
+            
+        ),
+        ```
+
+        Here we assume WSO2 IS is running in localhost on 9443.
+
+9.  Add the identity provider metadata.
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi metadata/saml20-idp-remote.php 
+    ```
+
+    1.  Add the following section to the file and save.
+
+        ``` java
+        $metadata['https://localhost:9443/samlsso'] = array(  
+            
+            'name' => array(  
+            
+            'en' =>  'WSO2 IS',  
+            
+            'no' =>  'WSO2 IS',  
+            
+        ),  
+            
+            'description'   =>  'Login with WSO2 IS SAML2 IdP.',  
+            
+            'SingleSignOnService'  =>  'https://localhost:9443/samlsso',  
+            
+            'SingleLogoutService'  => 'https://localhost:9443/samlsso',  
+            
+            'certFingerprint'      => '6bf8e136eb36d4a56ea05c7ae4b9a45b63bf975d'  
+            
+        );
+        ```
+
+    2.  Note that metadata \['
+        `                           https://localhost:9443/samlsso                         `
+        '\] should match the value of 'idp' in step 8.
+
+    3.  Note that "6bf8e136eb36d4a56ea05c7ae4b9a45b63bf975d" is the
+        thumbprint of the default certificate ships with WSO2 IS. SAML2
+        Response is signed with this certificate.
+
+## Configure the service provider in WSO2 IS
+
+1.  [Start WSO2 Identity Server](insert-link). 
+
+2.  Sign in. Enter your username and password to log on to the
+    [Developer Portal](https://localhost:9443/developer-portal).
+
+3.  Select **Applications** from the left hand side panel. 
+
+4.  Click on **New Application**.
+
+5.  Click on **SAML Web Application**.
+
+6.  Fill in the **Name** and optionally, provide a brief
+    **Description** of the application.  
+
+7.  Click **Next**.
+
+8.  Enter the **issuer ID** as simplesaml
+
+9.  Enter the **assertion consumer URL** as `http://localhost/simplesaml/module.php/saml/sp/saml2-acs.php/wso2-sp`.
+
+10. Click **Next**. 
+
+11. Check your configurations and click **Finish**. 
+
+12. Navigate to the **Access** tab of your newly created application.
+
+13. Enable **Single Logout Profile**. 
+
+14. Enter the **Single logout response URL** as `http://localhost/simplesamlphp/www/module.php/saml/sp/saml2-logout.php/wso2-sp`
+
+15. Click **Update**.
+
+## Configure a resident identity provider in WSO2 IS
+
+
+### Test SimpleSAMLphp
+
+1.  Go to <http://localhost/simplesaml> and then to " **Authentication**
```

Comment:
Go to --> Access

Reply:
"**Authentication**" --> **Authentication** 

" **Test configured authentication sources** " --> **Test configured authentication sources**